jdi-light/src/main/java/com/epam/jdi/light/asserts/core/DataListAssert.java

```diff
@@ -155,7 +154,7 @@ private boolean isDisplayed(T element) {
      */
     @JDIAction("Assert that '{name}' is not empty")
     public DataListAssert<T, D> notEmpty() {
-        jdiAssert(element.isEmpty() ? "list is empty" : "list is not empty", Matchers.is("list is not empty"));
+        jdiAssert(element().isEmpty() ? "list is empty" : "list is not empty", Matchers.is("list is not empty"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5600247)